xknx/management/procedures.py

```diff
@@ -106,7 +106,7 @@ async def nm_individual_address_read(
     return addresses
 
 
-async def nm_invididual_address_write(
+async def nm_individual_address_write(
```

Comment:
oops. Shall we add a `nm_invididual_address_write = nm_individual_address_write` line with some comment to avoid a breaking change for that?